vite.config.js

```diff
@@ -21,6 +21,11 @@ function LocaleHotReload() {
 const config = {
     plugins: [sveltekit(), LocaleHotReload()],
     build: { chunkSizeWarningLimit: 1600 },
+    server: {
```

Comment:
While testing, I encountered a `The request url "/Users/annieli/Development/wordplay/functions/src/shared/index.ts" is outside of Vite serving allow list.` error when I tried to navigate to the login page. This new entry fixes that.